app/controllers/users_controller.rb

```diff
@@ -100,6 +96,11 @@ def start_verification(to, channel = 'sms')
                           .verifications
                           .create(to:, channel:)
     verification.sid
+    current_user.verified = false
+  rescue
+    current_user.verified = true
```

Comment:
Why we are setting verified as true in case of error?
Also it will be better to catch a specific kind of errors.